website/license.html

```diff
@@ -6,6 +6,7 @@
   <head>
     <title>quick-lint-js license information</title>
     <meta charset="utf-8" />
+    <meta name="description" content="quick-lint-js license information."/>
```

Comment:
Minor: This description just repeats the title. Is it worth including?

Reply:
yeah, i don't think so. It's pretty self-explanatory.